packages/office-addin-debugging/src/commands.ts

```diff
@@ -86,7 +68,7 @@ export async function start(manifestPath: string, appType: string | undefined, c
             throw new Error("Please specify the application type to debug.");
         }
 
-        await startDebugging(manifestPath, appTypeToDebug, app, debuggingMethod, sourceBundleUrlComponents,
+        await startDebugging(manifestPathToDebug, appTypeToDebug, app, debuggingMethod, sourceBundleUrlComponents,
```

Comment:
Please don't rename `manifestPath`. The addition of "ToDebug" doesn't really add any meaning.